### ISSUE TITLE
What to change to use Cross entropy 

### ISSUE THREAD

**Issue by Oussamab21** [kimhc6028/soft-decision-tree#10]
Thank you for your code I was wondering what I should change to use the nn.CrossEntropy loss instead of the loss stated in the paper ?

Thank you
